<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detect</class>
 <widget class="QMainWindow" name="Detect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UFellWalker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>UFellWalker</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>n_dilation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ndilation">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>region_pixels_min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>thresh</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>core_pixels</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>peak_delta</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>nerest_dist</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pixelsmin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_thresh">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_corepixels">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>50</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_peakdelta">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nearestdist">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>7</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Facet</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>kopen_radius_first</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>kopen_radius_second</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>axis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>label_area_max</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>label_area_min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>region_pixels_min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>thresh_first</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>thresh_second</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>bins</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>keig_bins</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>kgra_bins</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kopenfirst">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kopensecond">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_axis">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_areamax">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>20</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_areamin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pixelsminF">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_threshfirst">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_threshsecond">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bins">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>11</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_keigbins">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kgrabins">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Cluster</string>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>gradmin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>rhomin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>deltamin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>volmin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>rms</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
     </property>
     <property name="text">
      <string>sigma</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_gradmin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0.01</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rhomin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1.5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deltamin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_volmin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>64</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rms">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0.4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sigma">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>0.8</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
